api/src/main/java/org/openmrs/module/commonreports/reports/DRCARTAbandonmentReportManager.java

```diff
@@ -0,0 +1,271 @@
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.openmrs.module.reporting.cohort.definition.BaseObsCohortDefinition.TimeModifier;
+import org.openmrs.module.reporting.cohort.definition.AgeCohortDefinition;
+import org.openmrs.module.reporting.common.DateUtil;
+
+@Component
+public class DRCARTAbandonmentReportManager extends ActivatedReportManager {
+	
+	@Autowired
+	private InitializerService inizService;
+	
+	@Override
+	public boolean isActivated() {
+		//return inizService.getBooleanFromKey("report.drc.hivStage.active", false);
+		return true;
+		
+	}
+	
+	@Override
+	public String getVersion() {
+		return "1.0.0-SNAPSHOT";
+	}
+	
+	@Override
+	public String getUuid() {
+		return "b0e1e4a6-749a-4741-b13c-4f4644c9df91";
+	}
+	
+	@Override
+	public String getName() {
+		return MessageUtil.translate("commonreports.report.drc.artAbandonment.reportName");
+	}
+	
+	@Override
+	public String getDescription() {
+		return MessageUtil.translate("commonreports.report.drc.artAbandonment.reportDescription");
+	}
+	
+	private Parameter getReportingDateParameter() {
+		String today = new SimpleDateFormat("yyyy-MM-dd").format(new Date());
+		return new Parameter("onOrBefore", MessageUtil.translate("commonreports.report.util.reportingEndDate"), Date.class,
+		        null, DateUtil.parseDate(today, "yyyy-MM-dd"));
+	}
+	
+	public static String col1 = "";
+	
+	public static String col2 = "";
+	
+	public static String col3 = "";
+	
+	public static String col4 = "";
+	
+	public static String col5 = "";
+	
+	public static String col6 = "";
+	
+	public static String col7 = "";
+	
+	public static String col8 = "";
+	
+	public static String col9 = "";
+	
+	public static String col10 = "";
+	
+	public static String col11 = "";
+	
+	@Override
+	public List<Parameter> getParameters() {
+		List<Parameter> params = new ArrayList<Parameter>();
+		params.add(getReportingDateParameter());
+		
+		return params;
+	}
+	
+	@Override
+	public ReportDefinition constructReportDefinition() {
+		ReportDefinition rd = new ReportDefinition();
+		rd.setUuid(getUuid());
+		rd.setName(getName());
+		rd.setDescription(getDescription());
+		rd.setParameters(getParameters());
+		
+		// artAbandonment Grouping
+		CohortCrossTabDataSetDefinition artAbandonment = new CohortCrossTabDataSetDefinition();
+		artAbandonment.addParameters(getParameters());
+		rd.addDataSetDefinition(getName(), Mapped.mapStraightThrough(artAbandonment));
+		
+		Map<String, Object> parameterMappings = new HashMap<String, Object>();
+		parameterMappings.put("onOrBefore", "${onOrBefore}");
+		
+		SqlCohortDefinition sqd = new SqlCohortDefinition();
+		
+		// ART plan -> Started drugs
+		// No Visit in past 90 days
+		// Not transferred in past 6 months (which would explain why missing in the 3months if done earlier)
+		// Refused ART in past 6 months (which would explain why missing in the 3months if done earlier)
```

Comment:
On these definitions here: why is there the 6 month timespan? Shouldn't transfer outs be taken into account as long as there's no newer transfer in and similarly with refusing ARTs?